src/pages/GameDisplayPage.js

```diff
@@ -56,3 +62,68 @@ const GameDisplayPage = () => {
 };
 
 export default GameDisplayPage;
+
+/**
+ * 勝者を返す関数
+ * @param {string[]} squares - 盤面を表す配列
+ * @param {number} n - 勝利条件（n個連続で並んだら勝ち）
+ * @returns {string} 勝者を返す
+ */
+function calculateWinner(squares, n) {
+  if (isConnectedN(squares, "O", n)) return "O";
+  if (isConnectedN(squares, "X", n)) return "X";
+  return null;
+}
+
+/**
+ * N個連続で並んでいるかどうか判定する関数
+ * @param {string[]} arr - 盤面を表す配列
```

Comment:
上のと同様に、二次元配列を示したほうが親切ですね！
```suggestion
 * @param {string[][]} arr - 盤面を表す二次元配列
```